backend/app/migrations/versions/cc7ec89e9135_added_jobs_types_table_and_foreign_key_.py

```diff
@@ -40,7 +40,7 @@ def upgrade():
     utcnow = datetime.utcnow()
 
     server_data = [{'id': 1, 'name': 'pdf', 'created_at': utcnow, 'updated_at': utcnow},
-                   {'id': 2, 'name': 'distribution-preview', 'created_at': utcnow, 'updated_at': utcnow}]
+                   {'id': 2, 'name': 'web-hosting-preview', 'created_at': utcnow, 'updated_at': utcnow}]
```

Comment:
Same as above. We can't change past migrations like this. To change the name, you'd need to add a new migration.